CloudFormation/Linux/amazon-linux-2023-attestable-image/base-template.yaml

```diff
@@ -0,0 +1,749 @@
+                - '  --architecture "$ARCH" \'
+                - '  --root-device-name "/dev/xvda" \'
+                - '  --virtualization-type hvm \'
+                - '  --ena-support \'
+                - '  --boot-mode uefi \'
+                - '  --tpm-support v2.0 \'
+                - '  --block-device-mappings "DeviceName=/dev/xvda,Ebs={SnapshotId=$SNAPSHOT_ID}" \'
+                - '  --tag-specifications "$EC2_TAG_SPECS" \'
+                - '  --query ''ImageId'' \'
+                - '  --output text)'
+                - 'echo $IMAGE_ID'
+
+  ImageWorkflow:
+    Type: AWS::ImageBuilder::Workflow
+    Properties:
+      Name: !Sub "${ResourceNamePrefix}-build-image-with-coldsnap"
+      Description: Creates AMI from a raw disk image using coldsnap.
+      Type: BUILD
+      Version: !Ref ImageBuilderResourceVersion
+      Data: !Sub |
+        name: ${ResourceNamePrefix}-build-image-with-coldsnap
+        description: Creates AMI from a raw disk image using coldsnap.
+        schemaVersion: 1.0
+        parameters:
+          - name: ImageNamePrefix
+            type: string
+            default: attestable-ami
+            description: Prefix for the AMI name
+        steps:
+          - name: LaunchBuildInstance
+            action: LaunchInstance
+            onFailure: Abort
+            inputs:
+              waitFor: "ssmAgent"
+          - name: ApplyBuildComponents
+            action: ExecuteComponents
+            onFailure: Abort
+            inputs:
+              instanceId.$: "$.stepOutputs.LaunchBuildInstance.instanceId"
+          - name: CreateOutputAmi
+            action: RunCommand
+            onFailure: Abort
+            inputs:
+              documentName: "${CreateOutputAmiDocument}"
+              instanceId.$: "$.stepOutputs.LaunchBuildInstance.instanceId"
+              parameters:
+                imageNamePrefix.$:
+                  - "$.parameters.ImageNamePrefix"
+                rawImagePath:
+                  - "/disk_image.raw"
+                tagSpecificationsFilePath:
+                  - "/tag_specification.json"
+          - name: TerminateBuildInstance
+            action: TerminateInstance
+            onFailure: Continue
+            inputs:
+              instanceId.$: "$.stepOutputs.LaunchBuildInstance.instanceId"
+        outputs:
+          - name: "ImageId"
+            value: "$.stepOutputs.CreateOutputAmi.output[0]"
+
+Outputs:
+  InfrastructureConfigurationArn:
+    Value: !GetAtt InfrastructureConfiguration.Arn
+    Export:
+      Name: !Sub "${AWS::StackName}-InfrastructureConfigurationArn"
+  ImageBuilderExecutionRoleArn:
+    Value: !GetAtt ImageBuilderExecutionRole.Arn
+    Export:
+      Name: !Sub "${AWS::StackName}-ImageBuilderExecutionRoleArn"
+  ImageWorkflowArn:
+    Value: !Sub "arn:${AWS::Partition}:imagebuilder:${AWS::Region}:${AWS::AccountId}:workflow/build/${ResourceNamePrefix}-build-image-with-coldsnap/x.x.x"
```

Comment:
With the same recent release referenced elsewhere, we could change these custom constructing of the latest ARN to use `!GetAtt Resource.LatestVersion.Arn`

[Reference for AWS::ImageBuilder::Workflow](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-imagebuilder-workflow.html#aws-resource-imagebuilder-workflow-return-values).